docs/core/EigenPodManager.md

```diff
@@ -266,6 +266,7 @@ For the validator whose balance should be updated, the caller must supply:
 
 *Requirements*:
 * Pause status MUST NOT be set: `PAUSED_EIGENPODS_VERIFY_BALANCE_UPDATE`
+* If the validator is withdrawable (withdrawable epoch is set), balance being proven MUST NOT be zero
```

Comment:
the balance is only zero in the case where the validator has become exited (i.e. the "withdrawal has completed"), right?
Two thoughts here:
1) want to be sure there isn't some weird edge case where a validator gets slashed down to actual zero and pushing an update + proving a withdrawal are _both forbidden at the same time_
2) I feel like the intention of this requirement should be clarified here, because otherwise it kinda just seems like a weird quirk / raises the question of "why"

Reply:
Good notes. I can fix the latter one now. 

The former, yes - good thing to raise to auditors, and hopefully some work improving our testing can make us more assured that this can't happen.

Reply:
Question on this -- is it possible for a validator's balance to reach 0 without having the "withdrawable epoch" set?

Because if so, the check this requirement references prevents `verifyBalanceUpdate` from functioning, maybe.

Reply:
> Question on this -- is it possible for a validator's balance to reach 0 without having the "withdrawable epoch" set?
> 
> Because if so, the check this requirement references prevents `verifyBalanceUpdate` from functioning, maybe.

I'm not personally sure. will page @gpsanant and @Sidu28 on this